client/src/components/admin/index.js

```diff
@@ -1,34 +1,57 @@
-import React from 'react'
+import React, { Component } from 'react'
 import { MuiThemeProvider, getMuiTheme } from 'material-ui/styles'
 import PropTypes from 'prop-types'
+import { Tabs, Tab, Paper } from 'material-ui'
 
 import UserTable from '../../containers/admin/userTable'
 import OrganizationTable from '../../containers/admin/organizationTable'
+import ProgramTable from '../../containers/admin/programTable'
 
 import './style.css'
 
 const muiTheme = getMuiTheme({
   palette: {
-    primary1Color: '#bed62f'
-  },
-  menuItem: {
-    hoverColor: '#bed62f'
+    primary1Color: '#ea4e46'
   }
 })
 
-const AdminPage = props => {
-  const { isSuperAdmin } = props
+class AdminPage extends Component {
+  componentWillMount() {
+    const { fetchUserPermissions } = this.props
 
-  return (
-    <MuiThemeProvider muiTheme={muiTheme}>
-      <h1 className="admin-title">Admin Center</h1>
-      {isSuperAdmin && <UserTable />}
-      <OrganizationTable />
-    </MuiThemeProvider>
-  )
+    fetchUserPermissions()
+  }
+
+  render() {
+    const { isSuperAdmin } = this.props
+
+    return (
+      <MuiThemeProvider muiTheme={muiTheme}>
+        <div className="table-container">
```

Comment:
Prepend this class name with "admin" to prevent any future collisions with any css classes.